libjsonutils/test/CMakeLists.txt

```diff
@@ -1,25 +1,36 @@
-# Download and unpack googletest at configure time
-configure_file(${CMAKE_SOURCE_DIR}/cmake/GoogleTest-CMakeLists.txt.in ${CMAKE_BINARY_DIR}/googletest-download/CMakeLists.txt)
-execute_process(COMMAND ${CMAKE_COMMAND} -G "${CMAKE_GENERATOR}" .
-  RESULT_VARIABLE result
-  WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/googletest-download )
-if(result)
-  message(FATAL_ERROR "CMake step for googletest failed: ${result}")
-endif()
-execute_process(COMMAND ${CMAKE_COMMAND} --build .
-  RESULT_VARIABLE result
-  WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/googletest-download )
-if(result)
-  message(FATAL_ERROR "Build step for googletest failed: ${result}")
-endif()
+# see /opt/local/share/cmake-3.16/Modules/FindGTest.cmake
+find_package(GTEST QUIET)
+
+if(NOT GTEST_FOUND)
+  # Download and unpack googletest at configure time
```

Comment:
I think `FetchContent` is the modern way to do this now. I suggest asking on the Slack channel for the recommended way these days.

Reply:
yes, this might be already quite old, these days I don't even now if gtest must be downloaded and compiled like this, I get mine from conan today. Even so, I would consider this a topic of another pull request

Reply:
I have planed a second patch using FetchContent